foodalert/static/foodalert/js/base.js

```diff
@@ -12,6 +14,8 @@ window.vueData.components = {
     "not-found-template": NotFoundTemplate,
     "unauthz-template": UnauthzTemplate,
     "welcome-template": WelcomeTemplate,
+    "close-template": CloseTemplate,
```

Comment:
Not sure if this is needed as it shows up elsewhere.

Reply:
Removed in the last commit